include/envoy/network/filter.h

```diff
@@ -160,7 +222,15 @@ class FilterChainFactory {
    * @return true if filter chain was created successfully. Otherwise
    *   false, e.g. filter chain is empty.
    */
-  virtual bool createFilterChain(Connection& connection) PURE;
+  virtual bool createNetworkFilterChain(Connection& connection) PURE;
+
+  /**
+   * Called to create the listener filter chain.
+   * @param listener supplies the listener to create the chain on.
+   * @return true if filter chain was created successfully. Otherwise
+   *   false.
```

Comment:
nit: reflow comment

Reply:
Not exactly sure what you mean here, but I moved the last word to the end of the previous line.